app/src/main/kotlin/com/intive/picover/auth/repository/AuthRepository.kt

```diff
@@ -80,6 +81,6 @@ class AuthRepository @Inject constructor(
 		return userProfile()
 	}
 
-	private fun requireUser() =
+	private fun requireUser(): FirebaseUser =
```

Comment:
Is an explicit type here needed? Why was it added `FirebaseUser`?

Reply:
I think it was added when I converted to the block body when I worked on the code. I overlooked it when I returned it to the previous version. Thank you.